TESTING.md

```diff
@@ -0,0 +1,107 @@
+# OSCAL Testing Standards
```

Comment:
Minor, non-blocking: I under this might turn into an input or an OSCAL artifact itself, but in this context we for metaschema-java?

```suggestion
# Testing Standards for metaschema-java
```

So why `for $PROJECT` is to make it look consistent and reuse it in liboscal-java and oscal-cli (I will venture a guess and say I want to align those projects with this one as well😄) and make the diff cleaner as we ramp this up.

Reply:
```suggestion
# Testing Standards
```